pipelines/main/platforms/build_linux.pgo_lto.arches

```diff
@@ -0,0 +1,8 @@
+# ROOTFS_IMAGE_NAME    TRIPLET                        ARCH           ARCH_ROOTFS    USE_JULIA_PGO_LTO   MAKE_FLAGS                                                                              TIMEOUT       ROOTFS_TAG    ROOTFS_HASH
+llvm_passes          x86_64-linux-gnu               x86_64         x86_64         PGO_LTO    JL_CFLAGS=-Werror -Wno-error=unused-function,JL_CXXFLAGS=-Werror -Wno-error=unused-function                                                   .             v6.00         0badf5d8794e21a03ac2ff2d46c1ab538ac02571
```

Comment:
It may be better to address the warning rather than ignoring it.

Reply:
Was a bug in clang, workaround here: https://github.com/JuliaLang/julia/pull/53897.